test/TestApp.AspNetCore.3.1/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="ErrorController.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TestApp.AspNetCore._3._1.Controllers
```

Comment:
Ah yea, all the `TestApp.AspNetCore.3.1` and `TestApp.AspNetCore.6.0` directories and all files can be deleted now. We just have one directory now `TestApp.AspNetCore`. The instrumentation now only supports .NET 6+ so there was no longer a need for separate test app projects.